lib/service/service.ts

```diff
@@ -12,15 +12,21 @@ const defaultOptions = {
  * Base class for all services
  * @memberof module:service
  */
-export = module.exports = class Service extends EventEmitter {
+export class Service extends events.EventEmitter {
+  public logger: any
+  public opened: boolean
+  public running: boolean
+  public servers: any
+  public pool: any
+
```

Comment:
Is this an intentional decision to make all these properties `public` on the various classes? Shouldn't we stay a bit more conservative here?

Reply:
Ah sorry!  Yes, agree that's a better default.